Production/scripts/ShuffleMergeLaw/analysis/tasks.py

```diff
@@ -0,0 +1,64 @@
+## see https://github.com/riga/law/tree/master/examples/htcondor_at_cern
+
+import six
+import law
+import subprocess
+import os
+import re
+
+from analysis.framework import Task, HTCondorWorkflow
+import luigi
+
+class ShuffleMergeSpectral(Task, HTCondorWorkflow, law.LocalWorkflow):
+  ## '_' will be converted to '-' for the shell command invocation
+  cfg               = luigi.Parameter(description = 'configuration file with the list of input sources')
+  input_path        = luigi.Parameter(description = 'input path with tuples for all the samples')
+  output_path       = luigi.Parameter(description = 'output directory')
+  pt_bins           = luigi.Parameter(description = 'pt bins')
+  eta_bins          = luigi.Parameter(description = 'eta bins')
+  mode              = luigi.Parameter(description = 'merging mode: MergeAll or MergePerEntry')
+  input_spec        = luigi.Parameter(description = '')
+  tau_ratio         = luigi.Parameter(description = '')
+  disabled_branches = luigi.Parameter(description = 'disabled-branches list of branches to disabled in the input tuples',
+                                      default = '')
+  n_threads         = luigi.IntParameter(description = 'number of threads', default = 1)
+  exp_disbalance    = luigi.IntParameter(description = '')
+  n_jobs            = luigi.IntParameter(description = 'number of HTCondor jobs to run')
+  start_entry       = luigi.FloatParameter(description = 'starting point')
+  end_entry         = luigi.FloatParameter(description = 'ending point')
+  
+  def create_branch_map(self):
+    step = 1. * (self.end_entry - self.start_entry) / self.n_jobs
+    return {i: (round(self.start_entry + i*step, 4), round(self.start_entry + (i+1)*step, 4)) for i in range(self.n_jobs)}
+
+  def output(self):
+    return self.local_target("stdout_stderr_job_{}.txt".format(self.branch))
```

Comment:
here you define a single file for stdout and stderr, but in framework.py there are two files and they have different names. Please, try to have a consistent logging structure.

Reply:
yes, the task output is the same of the framework output (which is the stdout of the condor job), so one of the two is redundant. I will remove it from the task.